hera_mc/data/test_data/initialization_data_apriori_antenna.csv

```diff
@@ -1,4 +1,4 @@
 antenna,start_gpstime,status
-HH0,1214481618,not_connected
-HH0,1214482618,not_connected
-HH1,1214482618,known_bad
+HH700,1214481618,not_connected
+HH700,1214482618,not_connected
+HH701,1214482618,known_bad
```

Comment:
Shouldn't every hooked up antenna have an entry in this table? Also, do you want 2 identical lines?

Reply:
It isn't needed.  My view for the test database is that it should be the minimum to fully test.

Reply:
ok. Is there no code that checks that all connected antennas have an entry in the apriori_antenna table?

Reply:
No. We could raise an issue if we think important -- I'm not sure it is.

Reply:
I guess it depends on how that table is used. What should the correlator & RTP do with antennas that are hooked up but don't have an entry in that table?